src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -146,6 +147,9 @@
       <!-- Temporarily skip the common files -->
       <FilteredContentFilesToPack Include="@(ContentFilesToPack)" Condition="'%(RecursiveDir)' != ''" />
       <None Include="@(FilteredContentFilesToPack)" PackagePath="content\%(RecursiveDir)%(Filename)%(Extension)" Pack="true" />
+      <Content Include="$(MicrosoftWebXdtExtensionsPath)"
```

Comment:
Is it necessary to have the condition since you added the error?

Reply:
No, could've removed that when I copy/pasted - doing that now